docs/manual/docs/cmd/graph/schemaextension/schemaextension-list.md

```diff
@@ -0,0 +1,44 @@
+# graph schemaextension list
+
+Get a list of schemaExtension objects created in the current tenant, that can be InDevelopment, Available, or Deprecated.
+
+## Usage
+
+```sh
+graph schemaextension list [options]
+```
+
+## Options
+
+Option|Description
+------|-----------
+`--help`|output usage information
+`-s, --status [status]`|The status to filter on
+`--owner [owner]`|The id of the owner to filter on
+`-p, --pageSize [pageSize]`|Number of objects to return
+`-n, --pageNumber [pageNumber]`|Page number to return if pageSize is specified (first page is indexed as value of 0)
+`--query [query]`|JMESPath query string. See [http://jmespath.org/](http://jmespath.org/) for more information and examples
+`-o, --output [output]`|Output type. `json,text`. Default `text`
+`--pretty`|Prettifies `json` output
+`--verbose`|Runs command with verbose logging
+`--debug`|Runs command with debug logging
+
+## Examples
+
+Get a list of schemaExtension objects created in the current tenant, that can be InDevelopment, Available, or Deprecated.
+
+```sh
+graph schemaextension list 
+```
+
+Get a list of schemaExtension objects created in the current tenant, with owner 617720dc-85fc-45d7-a187-cee75eaf239e
+
+```sh
+graph schemaextension list --owner 617720dc-85fc-45d7-a187-cee75eaf239e
+```
+
+## Additional information
+
+pageNumber is specified as a 0-based index. A value of 2 returns the third page of items. 
```

Comment:
Please move this to *remarks* (see: https://pnp.github.io/office365-cli/cmd/aad/approleassignment/approleassignment-list/)

Reply:
Thanks a lot @appieschot for your review 😊 I will work on them soon!

Reply:
Whoopsie daisies, this was ignored for long will be picking up this week folks. Thanks for your patience.

Reply:
Hey @garrytrinder  I have heaps of changes from the master to go into this branch.
What is the best way to safely get all the changes to the feature branch.

Reply:
The “simplest”, should be to update your local master branch with all the latest updates first, checkout your feature branch and then perform a rebase against your local master branch, `git rebase master`.

Reply:
Thanks Garry, yep tried this but it kept complaining about something, I might give this another go again tonight.